lighthouse-core/report/v2/renderer/details-renderer.js

```diff
@@ -55,6 +61,28 @@ class DetailsRenderer {
   }
 
   /**
+   * @param {!DetailsRenderer.DetailsJSON} text
+   * @return {!Element}
+   */
+  _renderURL(text) {
+    const element = this._renderText(text);
+    element.classList.add('lh-text__url');
+    return element;
+  }
+
+  /**
+   * @param {!DetailsRenderer.DetailsJSON} obj
+   * @return {!Element}
+   */
+  _renderThumbnail(obj) {
+    const element = this._dom.createElement('img', 'lh-thumbnail');
+    element.src = obj.text.url;
+    // ignore obj.text.mimeType
+    element.alt = 'Image preview';
```

Comment:
Should we also add `element.title = obj.text.url;` so there's a full url when users hover over the image?